notes-docs.md

```diff
@@ -0,0 +1,51 @@
+Philosophy: one thing to do things
+
+Questions:
+
+-   What to do about the egghead lessons? They have a reference the decorator
+    syntax.
+
+-   The README is rather long. Credits, testimonials, and sponsors in multiple
+    places.
```

Comment:
That's probably a matter of taste. Someone like succinct more categories docs and someone wants just skim through everything in one place. I don't think there is a "gold standard" in this.

Reply:
The job of the landing page is to sell people on MobX and to give a brief introduction about what it is all about. To support that job we shouldn't lose the reader's attention or have them be lost in details.

I think the credits, testimonial and multiple testimonial sections help sell people on MobX, as it's "hey people use it!". But there's quite a lot of it -- in particular there are *two* sponsor sections - one at the top and one at the end. At the end there are unrelated bits such as "contributing" mixed in.

I think the sponsors at the top works best to sell things -- we could put the other bits onto a separate page and link this prominently on the top, where they can actually carry *more* weight as those who are only interested in such things (say, managers) can click through right away and don't have to pass a lot of code.

Reply:
Well, this isn't a landing page to sell a product. People will most likely hear about MobX from other sources first and they will come here to learn technical details. Regular marketing tactics are weird for open source.

Let's say that sponsors are important to keep on top as they kinda "bought" such a prominent place there. Followed by installation and browser support (should be environment support). But the "Getting started" feels weird like next. It's just a spill of links. I think we can move it behind the Core concept section and rather let people have a quick peek at basics. Perhaps even merge those links with section "Further resources and documentation".

Reply:
Sounds like a good proposal @FredyC. Sponsors are primarily there because they paid, although it gives some credibility as well :). I think it would be fine to actually move everything else to separate pages (concepts, getting started, and environment support as part of installation?) if that doesn't blow up navigation too much, a bit like done with immer https://github.com/immerjs/immer (but let's leave the sponsors)

Reply:
Okay, I'm going to see about splitting things up into separate pages. Concerning the sponsors, there are three relevant sections - the section on the top, the backers section and *another* sponsors section at the bottom. Do we want to keep them all on the landing page or are we talking about the top section?

Reply:
I've split things into separate pages, making the landing page lighter, though there's still lot of stuff at the bottom.